### PR TITLE
util/set: migrate test-infra to testify

### LINKED ISSUE #26417: migrate test-infra to testify for `util/set` pkg

### PR DESCRIPTION
**Pull request by PragmaTwice**
### What problem does this PR solve?

Issue Number: close #26417 

Problem Summary: refer to #26022

### Release note 

- No release note

### CODE REVIEW

File: util/set/float64_set_test.go
Comment (tisonkun) [inline util/set/float64_set_test.go:35]:
```suggestion
	assert.Equal(len(vals), set.Count())
```

Be award of the expected and actual position. Well, it is always a problem for testing framework 🤣

Comment (tisonkun) [reply util/set/float64_set_test.go:35]: ditto to others

Comment (PragmaTwice) [reply util/set/float64_set_test.go]: Done via de3915dab5ab97e366bcf634c488a2e123c0f139

**Review by xhebox — APPROVED** [review]

**Review by tisonkun — APPROVED** [review]
LGTM

**Comment by tisonkun** [comment]
@PragmaTwice I remove these three reviewers as I think they can hardly review in time.

/cc @wjhuang2016 

PTAL. This pull request is still waiting for another valid approval (note that I'm not a reviewer yet), and also a committer to trigger merging.

**Review by wjhuang2016 — APPROVED** [review]

**Pull request merged**
